<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>500</height>
   </rect>
  </property>

  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">

    <!-- FORM DE CADASTRO -->
    <item>
     <widget class="QGroupBox" name="groupCadastro">
      <property name="title">
       <string>Novo Lançamento</string>
      </property>

      <layout class="QGridLayout" name="formLayout">

       <item row="0" column="0">
        <widget class="QLabel" name="labelDescricao">
         <property name="text">
          <string>Descrição:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="inputDescricao"/>
       </item>

       <item row="1" column="0">
        <widget class="QLabel" name="labelValor">
         <property name="text">
          <string>Valor:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="inputValor">
         <property name="maximum">
          <double>9999999.99</double>
         </property>
         <property name="decimals">
          <number>2</number>
         </property>
        </widget>
       </item>

       <item row="2" column="0">
        <widget class="QLabel" name="labelCusto">
         <property name="text">
          <string>Custo:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="inputCusto">
         <property name="maximum">
          <double>9999999.99</double>
         </property>
         <property name="decimals">
          <number>2</number>
         </property>
        </widget>
       </item>

       <item row="3" column="0">
        <widget class="QLabel" name="labelTipo">
         <property name="text">
          <string>Tipo:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="comboTipo">
         <item><property name="text"><string>Entrada</string></property></item>
         <item><property name="text"><string>Saída</string></property></item>
        </widget>
       </item>

       <item row="4" column="0">
        <widget class="QLabel" name="labelData">
         <property name="text">
          <string>Data:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDateEdit" name="inputData">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>

       <item row="5" column="1">
        <widget class="QPushButton" name="btnAdicionar">
         <property name="text">
          <string>Adicionar</string>
         </property>
        </widget>
       </item>

      </layout>
     </widget>
    </item>

    <!-- LISTA -->
    <item>
     <widget class="QListWidget" name="lista"/>
    </item>

    <!-- FILTRO -->
    <item>
     <widget class="QGroupBox" name="groupFiltro">
      <property name="title">
       <string>Filtrar</string>
      </property>

      <layout class="QGridLayout" name="filterLayout">

       <item row="0" column="0">
        <widget class="QLabel" name="labelFiltroInicio">
         <property name="text">
          <string>Data início:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDateEdit" name="filtroDataInicio">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>

       <item row="1" column="0">
        <widget class="QLabel" name="labelFiltroFim">
         <property name="text">
          <string>Data fim:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDateEdit" name="filtroDataFim">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>

       <item row="2" column="0">
        <widget class="QLabel" name="labelFiltroTipo">
         <property name="text">
          <string>Tipo:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="filtroTipo">
         <item><property name="text"><string>Todos</string></property></item>
         <item><property name="text"><string>Entrada</string></property></item>
         <item><property name="text"><string>Saída</string></property></item>
        </widget>
       </item>

       <item row="3" column="1">
        <widget class="QPushButton" name="btnFiltrar">
         <property name="text">
          <string>Filtrar</string>
         </property>
        </widget>
       </item>

      </layout>
     </widget>
    </item>

    <!-- BOTÃO SALVAR -->
    <item>
     <widget class="QPushButton" name="btnSalvar">
      <property name="text">
       <string>Salvar Arquivo</string>
      </property>
     </widget>
    </item>

   </layout>
  </widget>

 </widget>
 <resources/>
 <connections/>
</ui>
